challenges/objects-arrays.js

```diff
@@ -97,29 +164,31 @@ The zoos need a list of all their animal's names (names only, not scientific) co
 
 */
 
-const lowerCase = [];
+let lowerCase = zooAnimals.map((animal) => {
+  return animal.animal_name.toLowerCase();
+});
 console.log(lowerCase); 
 
 /* Request 3: .filter() 
 
 The zoos are concenred about animals with a lower population count. Find out which animals have a population less than 5.
 
 */
-const lowerPopulation = [];
+
+const lowerPopulation = zooAnimals.filter(lowerPopulation => lowerPopulation.zooanimals < 5);
```

Comment:
Currently, the output of the lowerPopulation array is 0. Nothing is added to the array through this filter.

The reason is because `lowerPopulation.zooanimals` doesn't exist.

`lowerPopulation.population` is a key in the array, so that does work. But the naming is still confusing. 

I suggest this:
```suggestion
const lowerPopulation = zooAnimals.filter(element => element.population < 5);
console.log(lowerPopulation);
```